poweriq-worker/src/main/java/com/vmware/flowgate/poweriqworker/jobs/PowerIQService.java

```diff
@@ -727,11 +773,8 @@ private void syncRealtimeData(FacilitySoftwareConfig powerIQ) {
          return;
```

Comment:
double check the getMappedAsset method. we should add the source filter and not hardcode with nlyte. 

so that we only query pdus related with this POwerIQ and Mapped.

Reply:
This  logic wille be updated in next work plan